build.gradle

```diff
@@ -9,8 +9,7 @@ buildscript {
         classpath 'com.github.maiflai:gradle-scalatest:0.10'
         classpath "gradle.plugin.org.scoverage:gradle-scoverage:2.0.0"
         classpath "gradle.plugin.edu.sc.seis.gradle:launch4j:1.6.1"
-        classpath "org.scalamock:scalamock-scalatest-support_2.11:3.2.2"
-
+        classpath 'org.scalamock:scalamock-scalatest-support_2.11:3.2.2'
```

Comment:
Do we really need scalaMock, as we know that is not working properly for our needs?